sdk/go/common/resource/properties_test.go

```diff
@@ -159,3 +159,12 @@ func TestCopy(t *testing.T) {
 	src["c"] = NewNumberProperty(99.99)
 	assert.Equal(t, 2, len(dst))
 }
+
+func TestSecretUnknown(t *testing.T) {
+	o := NewOutputProperty(Output{
```

Comment:
We should add tests for `Computed` as well. AFAIK `Output` values are pretty much unused; we generally use `Computed` instead.

Reply:
added